docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,274 @@
+
+* id: A unique identifier of the course; a serialized representation of the
+  opaque key identifying the course.
+
+* media: An object that contains named media items, including the following
+  objects.
+
+  * course_image: An image to show for the course.
+
+    * uri: The location of the image.
+
+  * course_video: A video about the course.
+
+    * uri: The location of the video.
+
+  * image: URLs for images in the course, including ``raw``, ``small``, or
+    ``large`` versions.
+
+* name: The name of the course.
+
+* number: The catalog number of the course.
+
+* org: The name of the organization that owns the course.
+
+* overview: An HTML textual description of the course.
+
+* short_description: A textual description of the course.
+
+* start: The date on which the course begins.
+
+* start_display: The start date of the course, formatted for reading.
+
+* start_type: Indicates how ``start_display`` was set. Possible values are:
+
+  * "string": Manually set.
+  * "timestamp": Generated from the ``start`` timestamp.
+  * "empty": The start date is not shown.
+
+* pacing: The type of pacing for this course. Possible values are
+  ``instructor`` and ``self``.
+
+* course_id: The course key. This field might be returned but is deprecated.
+  You should use ``id`` instead.
+
+
+==============================================================
+Example Response Showing The List of Courses Visible to a User
+==============================================================
+
+.. code-block:: json
+
+    {
+
+     "blocks_url":"/api/courses/v1/blocks/?course_id=edX%2Fexample%2F2012_Fall",
+     "media": {
+        "course_image": {
+        "uri": "/c4x/edX/example/asset/just_a_test.jpg",
+        "name": "Course Image"
+                        }
+              },
+      "description": "An example course.",
+      "end": "2015-09-19T18:00:00Z",
+      "enrollment_end": "2015-07-15T00:00:00Z",
+      "enrollment_start": "2015-06-15T00:00:00Z",
+      "course_id": "edX/example/2012_Fall",
+      "name": "Example Course",
+      "number": "example",
+      "org": "edX",
+      "start": "2015-07-17T12:00:00Z",
+      "start_display": "July 17, 2015",
+      "start_type": "timestamp"
+    }
+
+
+.. _Get the details for a course:
+
+*************************
+Get Details for a Course
+*************************
+
+The endpoint to get the details for a course is
+``/api/courses/v1/courses/{course_key}/``.
+
+=====================
+Use Case
+=====================
+
+Get the details for a course that you specify using a course key.
+
+=====================
+Example Requests
+=====================
+
+GET /api/courses/v1/courses/edX%2FDemoX%2FDemo_Course
+
+=====================
+Query Parameters
+=====================
+
+* username (optional) - The username of the user for whom the course data is
+  being accessed. If the request is made by an anonymous user, username is not
+  required.
+
+=====================
+Response Values
```

Comment:
Same comments as above.  Consider referring readers to the endpoint above to reduce duplication of text.